chain/sync.go

```diff
@@ -634,18 +657,19 @@ func (syncer *Syncer) ValidateBlock(ctx context.Context, b *types.FullBlock) err
 		return nil
 	})
 
-	eproofCheck := async.Err(func() error {
-		if err := syncer.VerifyElectionPoStProof(ctx, h, waddr); err != nil {
-			return xerrors.Errorf("invalid election post: %w", err)
-		}
-		return nil
-	})
+	//eproofCheck := async.Err(func() error {
+	//if err := syncer.VerifyElectionPoStProof(ctx, h, waddr); err != nil {
+	//return xerrors.Errorf("invalid election post: %w", err)
+	//}
+	//return nil
+	//})
```

Comment:
We can still check WinningPoSt in parallel here, right?

Reply:
yup